File: OPENAPI-REST-API/openapi-client/clojure/src/inter_server_management_api/specs/vps_order_location_stock_1.clj
(ns inter-server-management-api.specs.vps-order-location-stock-1
  (:require [clojure.spec.alpha :as s]
            [spec-tools.data-spec :as ds]
            )
  (:import (java.io File)))


(def vps-order-location-stock-1-data
  {
   (ds/opt :kvm) boolean?
   (ds/opt :kvmstorage) boolean?
   (ds/opt :hyperv) boolean?
   })

(def vps-order-location-stock-1-spec
  (ds/spec
    {:name ::vps-order-location-stock-1
     :spec vps-order-location-stock-1-data}))

File: OPENAPI-REST-API/swagger-client/scala-akka-http-server/src/main/scala/io/swagger/server/model/VpsOrder_locationStock_1.scala
package io.swagger.server.model


/**
 * @param kvm  for example: ''true''
 * @param kvmstorage  for example: ''true''
 * @param hyperv  for example: ''true''
 */
case class VpsOrder_locationStock_1 (
  kvm: Option[Boolean],
  kvmstorage: Option[Boolean],
  hyperv: Option[Boolean]
)

